README.MD

```diff
@@ -14,13 +14,7 @@ Total message size (bytes): Payload + 3
 
 #### Install Arduino Libraries
 
-Copy the `src\SerialProtocol` folder to:
-
-- Windows: `%USERPROFILE%\Victor\Documents\Arduino`
-- Linux: `~/sketchbook/libraries` or `/usr/share/arduino/libraries`
-- Mac: `Documents/Arduino/libraries`
-
-Note: You must restart the Arduino IDE if it was running before you copied the files.
+Download the Repository, Go to Sketch>Include Library>Add .zip Library> Select the Downloaded .zip File.
 
 #### Samples
 
```

Comment:
I don't think the `samples` folder exist anymore in your change, right? Can you also update this part of the readme?